README.md

```diff
@@ -354,20 +344,108 @@ The CLI accepts a series of files or globs to be formatted, as well as a set of
-| `verbose`  | `v`       | Print the output of the formatting                                                                                                                                                                                            | **false**                    |
-| `help`     | `h`       | Print help menu                                                                                                                                                                                                               |                              |
+Options can also be passed as environment variables.
+
+```
+FORMAT_PACKAGE_VERBOSE=true
+```
+
+| **Option** | **Alias** | **ENV**                | **Description**                                                                                                                                                                                                               | **Default**                  |
+| ---------- | --------- | ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
+| `config`   | `c`       | FORMAT_PACKAGE_CONFIG  | Path to a custom configuration to use. This configuration can be JavaScript, `JSON`, or any other format that your configuration of node can `require`. The default configuration can be found [here](lib/defaults/index.js). |                              |
+| `write`    | `w`       | FORMAT_PACKAGE_WRITE   | Write the output to the location of the found `package.json`                                                                                                                                                                  | **false**                    |
+| `ignore`   | `i`       | FORMAT_PACKAGE_IGNORE  | Patterns for ignoring matching files                                                                                                                                                                                          | **`['**/node_modules/**']`** |
+| `verbose`  | `v`       | FORMAT_PACKAGE_VERBOSE | Print the output of the formatting                                                                                                                                                                                            | **false**                    |
+| `help`     | `h`       |                        | Print help menu                                                                                                                                                                                                               |                              |
 
 You can also see the available options in the terminal by running:
 
 ```
 yarn format-package --help
 ```
 
+## Config
```

Comment:
I'm not sure where it should go, but I think we should probably add a small blurb letting developers know that the configuration can be partial.